tidy3d/web/core/account_core.py

```diff
@@ -0,0 +1,65 @@
+"""Tidy3d user Account."""
```

Comment:
Since this file is already in `web/core` it can just be called `account.py`?

Reply:
done